config/service_qos.go

```diff
@@ -91,12 +102,18 @@ var shannonQoSTypes = map[protocol.ServiceID]ServiceQoSType{
 // Reference: EVM chain IDs are sourced from https://chainlist.org
 var shannonEVMChainIDs = map[protocol.ServiceID]string{
 	// EVM service IDs
-	"eth": "0x1", // ETH Mainnet (1)
+	"eth": "0x1", // ETH Mainnet (1) [default EVM chain ID]
```

Comment:
For this and `cosmoshub-4`, I was actually thinking to USE the constant directly.

It's a special case, so why risk "copy-paste" errors?

Not going to block the PR but I strongly believe this is the more "code sustainable, less error-prone" way to move forward.